### DISCUSSION TITLE
help reading a very dirty csv

### DISCUSSION THREAD

**Discussion by djouallah** [chdb-io/chdb#205] [Q&A]
the csv don't have a fixed number of columns and could not make it works in chdb
https://colab.research.google.com/drive/1mLtirbBbTP34Ets83ZIYVdQV6fZprqIQ#scrollTo=MzY2UICwhNY7

**Comment by auxten** [comment]
See #206 
<img width="1279" alt="image" src="https://github.com/chdb-io/chdb/assets/240147/07a57a2c-1d72-4fee-9eb0-21bfad78c224">

https://colab.research.google.com/drive/15wTQ_JeDwyhO8_Zcc3UeGPDosnSGXU1r?usp=sharing
